api/agentpb/agent.proto

```diff
@@ -126,6 +127,12 @@ message QueryActionMap {
   map<string, QueryActionValue> map = 1;
 }
 
+// QueryActionBinary represents primitive.Binary used in query Actions.
+message QueryActionBinary {
+  int32 subtype = 1;
```

Comment:
What is the `subtype` ?

Reply:
please see https://bsonspec.org/spec.html 
`Generic binary subtype`